configuration/concepts/nutrition.csv

```diff
@@ -4,7 +4,7 @@ edaef9f4-2b5b-4b71-9019-74f1d40ad4d7,,CIEL:460,Oedema,Œdème,Oedema,Œdème,,,M
 8dfc3ec9-771a-42b6-ad45-aa0ac15b660c,,,MUAC<115mm,PB<115mm,MUAC<115mm,PB<115mm,,,Misc,N/A,,,,,,,,,,,
 6a7ea608-0546-40f1-92d2-cd020e980414,,CIEL:1655,Admission Criteria,Critère d'admission,Admission Criteria,Critère d'admission,,,Misc,Coded,,Oedema;Weight-for-height<-3SD/BMI<16;MUAC<115mm,,,,,,,,,
 a2224476-1827-4270-9ace-c8fd7898fd21,,,Readmission,Réadmission,Readmission,Réadmission,,,Misc,Coded,,Yes;No,,,,,,,,,
-fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,CIEL:163102,Weight goal,Objectif de poids,Weight goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
+fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,CIEL:163102,Weight Goal,Objectif de poids,Weight Goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,

```

Comment:
It seems that the decimals are allowed (form can be saved), but the UI suggests it's not:
<img width="393" alt="Screenshot 2020-10-22 at 20 15 39" src="https://user-images.githubusercontent.com/10692811/96913161-8fd11080-14a3-11eb-953e-88403cf6c21d.png">
Do you experience the same thing on your browser? This is also the case for Weight in Vital Signs form.

Reply:
Everything works out well on my end

Reply:
Fixed Historique de sante form too

Reply:
OK then. Thanks